src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -392,19 +392,20 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
     }
 
     private createInitiatorToevoegenMenuItems(): void {
-        if (this.zaak.acties.toevoegenInitiatorPersoon || this.zaak.acties.toevoegenInitiatorBedrijf) {
-            this.menu.push(new HeaderMenuItem('initiator.toevoegen'));
-            if (this.zaak.acties.toevoegenInitiatorPersoon) {
-                this.menu.push(new ButtonMenuItem('initiator.toevoegen.persoon', () => {
+        if (this.zaak.acties.toevoegenInitiatorPersoon || this.zaak.acties.toevoegenInitiatorBedrijf ||
+            this.zaak.acties.toevoegenBetrokkenePersoon || this.zaak.acties.toevoegenBetrokkeneBedrijf) {
+            this.menu.push(new HeaderMenuItem('betrokkenen'));
+            if (this.zaak.acties.toevoegenInitiatorPersoon || this.zaak.acties.toevoegenInitiatorBedrijf) {
```

Comment:
Als er nergens anders nog onderscheid wortdt gemaakt tussen toevoegenInitiatorPersoon en toevoegenInitiatorBedrijf dan heeft het geen zin om er 2 verschillende acties voor te hebben. Dus 
Of we maken dat verschil wel ergens. Dat betekend dat het ergens in het BetrokkeneZoekComponent moet gebeuren. 
Of we maken van deze 2 acties 1 actie: toevoegenInitiator

Reply:
Jij gaf aan dat je het liever gesplitst liet totdat het duidelijk was. ;-)

Deze change voegt de twee panels samen. Ik moet de twee tabs nog elk van een policy-check voorzien, zou ik zeggen. Maar als je ze samen gaat voegen dan hoef ik dat niet te doen.